File: Services/Medico.Api/ClientApp/src/app/_models/patientOrder.ts
export interface PatientOrder {
  patientId: string;
  appointmentId?: string;
  notes?: string;
  referenceNo?: string;
  physicianId: string;
  insuranceId?: string;
  vendorId: number;
  dateOrdered: Date;
  attachmentId?: number;
  patientOrderItems: PatientOrderItem[];
  userIds?: string[];
  reminderDate?: Date | null;
}

export interface PatientOrderItem {
  labTestId: any;
  quantity?: number;
}

File: Domain/Medico.Domain/Models/PatientOrder.cs
using System;
using System.Collections.Generic;

namespace Medico.Domain.Models
{
    public class PatientOrder: Entity
    {
        public Guid PatientId { get; set; }
        public string OrderNumber { get; set; }
        public string Notes { get; set; }
        public int? AttachmentId { get; set; }
        public string OrderStatus { get; set; }
        public Guid CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public string ReferenceNo { get; set; }
        public Guid PhysicianId { get; set; }
        public Guid InsuranceId { get; set; }
        public DateTime DateOrdered { get; set; }
        public int VendorId { get; set; }
        public string UserIds {get; set;}
        public IEnumerable<PatientOrderItem> PatientOrderItems { get; set; }
    }
}
